changelog.d/20211029_162341_kurtmckee_sdk_v3.rst

```diff
@@ -0,0 +1,15 @@
+Features
+--------
+
+-   Upgrade to Globus SDK v3.
+-   Add code linting, documentation build testing, and a bunch of unit tests.
+    (Code coverage of modules currently imported during testing is 73%.)
```

Comment:
I would characterize this as a different heading ("Changes" or "Other"? I don't know if these exist by default, at least in this project) since it's not user-facing

Reply:
Agreed. It's not difficult to add these, it just needs to be codified in the config. For the scriv config I only copied what I saw in past release announcements:

```
categories = ["Features", "Bugfixes", "Documentation"]
```

Adding a tag like "Other" seems good to me. @jimPruyne / @urimandujano?

Reply:
I've updated the changelog fragment but it's still worthwhile to consider the categories.

Reply:
I'm not sure what to call these -- maybe "Internal", "Development" or "Meta changes"? I don't have a strong opinion but I'm willing to go with whatever feels good to you two.